e2e_test/source/cdc/cdc.check_new_rows.slt

```diff
@@ -180,3 +192,17 @@ select id, my_bytea from list_with_null_shared order by id;
 2 {"\\x00","\\x01","\\x02"}
 3 {NULL,"\\x99","\\xaa"}
 4 {"\\x88","\\x99","\\xaa"}
+5 NULL
+6 NULL
+
+query TTTTTTT
+SELECT * FROM postgres_all_types order by c_boolean,c_bigint,c_date;
+----
+f -32767 -2147483647 -9223372036854775807 -10.0 -10000 -10000 (empty) \x00 0001-01-01 00:00:00 2001-01-01 00:00:00 2001-01-01 08:00:00+00:00 00:00:00 {} bb488f9b-330d-4012-b849-12adeb49e57e happy {f} {-32767} {-2147483647} {-9223372036854775807} {-10.0} {-10000} {-10000} {""} {"\\x00"} {0001-01-01} {00:00:00} {"2001-01-01 00:00:00"} {"2001-01-01 08:00:00+00:00"} NULL {"{}"} {bb488f9b-330d-4012-b849-12adeb49e57e} NULL
```

Comment:
The test case is too large to follow, we may need to split the `select *` into multiple small `select` statements for better readability.

Reply:
updated